### PR TITLE
getting the service health/status

### PR DESCRIPTION
**Pull request by prashantramangupta**
getting the service health/status by making the grpc call to the service endpoint using grpc standard proto. Resolves Issue #193

### CODE REVIEW

File: snet_cli/mpe_service_command.py [hunk 1]
```diff
@@ -171,6 +174,32 @@ def print_service_metadata_from_registry(self):
         metadata      = self._get_service_metadata_from_registry()
         self._printout(metadata.get_json_pretty())
 
+    def _service_status(self, url, secure=True):
+        channel = None
+        try:
+            channel = open_grpc_channel(endpoint=url)
+            stub = heartb_pb2_grpc.HealthStub(channel)
+            try:
```
Comment (raamb) [inline snet_cli/mpe_service_command.py]: Lets add a timeout for the healthcheck as well. We can go with 15 seconds

Comment (prashantramangupta) [reply snet_cli/mpe_service_command.py]: added timeout as 10 sec same as in Dapp

**Comment by astroseger** [comment]
1. How snet-cli will find health_pb2 and health_pb2_grpc? I would propose to put corresponding proto file in ```snet_cli/resources/proto```. In this case it will be compiled automatically and you will be able to import it (as ```snet_cli.resources.proto.<name_of_file>```)
2. It will not work for the service with multiply groups. You should add ```add_p_group_name(p)``` parameter.
3. So health service doesn't return any interesting information, so we can print only Available/Not Available?
4. I would add some CI tests. But it will require add heart beats in our dummy service... We could do it later...

**Comment by prashantramangupta** [comment]
@astroseger
1.  health_pb2 and health_pb2_grpc is part of the standard grpc package(grpcio-health-checking==1.19.0)
2. It is expected to give available/unavailable info. Let me know if you need more info here.
3. working on this 
**It will not work for the service with multiple groups. You should add add_p_group_name(p) parameter.**

**Comment by prashantramangupta** [comment]
Points are noted and work is in progress

**Pull request closed without merging**